app/routes/index.tsx

```diff
@@ -54,19 +64,37 @@ export default function Index(): JSX.Element {
 	return (
 		<>
 			<Navbar groupedPages={groupedPages} />
-			<Container w="full" maxW="7xl" mx="auto" py={8} centerContent>
-				<chakra.section>
+			<Container
+				w="full"
+				// bg="#f6f9fc"
+				bg={useColorModeValue("gray.50", "")}
+				maxW="full"
```

Comment:
Why not nest the previous `<Container>` in a full-width `chakra.section`? Setting a `100%` max-width on a Container kinda goes against the point of the component.